rustls/src/crypto/cipher.rs

```diff
@@ -124,6 +124,11 @@ impl Iv {
         iv.0.copy_from_slice(value);
         iv
     }
+
+    /// Return the `Iv`'s underlying byte slice, of precisely `NONCE_LEN` bytes.
+    pub fn value(&self) -> [u8; NONCE_LEN] {
```

Comment:
Does it make sense to copy here, or are we generally going to be passing this as a slice to the next API? Maybe this should yield a reference to the `Iv`'s field instead? And if so, maybe using `AsRef` for this makes sense?

Reply:
I switched to implementing `AsRef`.